index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <meta charset='UTF-8'/>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <div>
```

Comment:
This ```<div>``` tag is not necessary. Try not to wrap components in a div if there is no style apply to it.